src/unread/unreadSelectors.js

```diff
@@ -140,9 +140,15 @@ export const getUnreadStreamsAndTopics = createSelector(
       return totals;
     }, {});
 
-    const sortedStreams = Object.values(unreadMap)
-      .sort(caseInsensitiveCompareObjFunc('streamName'))
-      .sort((a: any, b: any): number => +b.isPinned - +a.isPinned);
+    const sortedStreams = Object.values(unreadMap).sort((a: any, b: any): number => {
+      // if both are pinned
```

Comment:
Do you need three lines for this comment? :)

Reply:
I liked this break down, easy to read 😅 
Should I move to one line?

Reply:
Yeah :)